pkg/common/oras/authprovider/azure/azureworkloadidentity.go

```diff
@@ -39,7 +39,8 @@ type azureWIAuthProvider struct {
 }
 
 type azureWIAuthProviderConf struct {
-	Name string `json:"name"`
+	Name     string `json:"name"`
+	ClientID string `json:"clientID"`
```

Comment:
I think this field should be optional --> add `omitempty` since it's not a required field in the chart either?

Reply:
thx and added